news/13156.bugfix.rst

```diff
@@ -0,0 +1 @@
+Switch to using lstat() instead of stat() to not match symlink targets.
```

Comment:
Could we focus on the impact for the end user? Something like "the shortened python command in the pip upgrade prompt is not emitted for unactivated virtual environments". Please use that idea, but please don't copy my wording :slightly_smiling_face:

Reply:
Done.